src/views/Search/Records.vue

```diff
@@ -1,13 +1,18 @@
 <template>
-    <div class="outputTable">
+    <div class="outputGrid" v-if="client">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces but found 4 spaces.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)